src/test/java/com/theironyard/war/GameTest.java

```diff
@@ -0,0 +1,32 @@
+package com.theironyard.war;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Nigel on 6/26/16.
+ */
+public class GameTest {
+    Player player1 = new Player("p1");
+    Player player2 = new Player("p2");
+
+    @Test
+    public void whenPlayersHandIsBigEnoughForWarReturnTrue() {
+        Game game = new Game(player1, player2);
+        boolean allowWar = game.allowWar();
+
+        assertFalse("Should return false if one or both player do not have enough cards for war. but returned true.", allowWar);
```

Comment:
This is literally testing the opposite of what the test says it should.  Additionally, there should be a sister test so that you test both that the function returns true and false appropriately.